Mittons.Fixtures.sln

```diff
@@ -1,24 +1,23 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+# Visual Studio Version 17
```

Comment:
I keep going back and fourth on this one. Did this update because you made a change using visual studio, or did visual studio require you to update it when you first loaded the project?

If it wasn't a forced thing when you loaded visual studio, I think I'd prefer to keep it at version 16 since that's what the dotnet template uses:
https://github.com/dotnet/templating/blob/main/template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

If you think that needs updated (which it may need), we should probably get a PR into their project to update the template, but until then I feel like we should go with what the dotnet commands generate instead of what the IDE's generate, unless there is a breaking issue.

Reply:
Going to close this, since it's not an issue for this ticket, but I think we should probably do a side conversation on this, and whether we _could_ submit a PR to that project I linked for the change. I'd like to get dotnet and vs in sync, so this doesn't come up again later!